src/actions/indexation/indexable-term-indexation-action.php

```diff
@@ -132,9 +132,21 @@ protected function get_query( $count, $limit = 1 ) {
 			"
 			SELECT $select
 			FROM {$this->wpdb->term_taxonomy}
-			WHERE term_id NOT IN (SELECT object_id FROM $indexable_table WHERE object_type = 'term') AND taxonomy IN ($placeholders)
+			WHERE term_id NOT IN (SELECT object_id FROM $indexable_table WHERE object_type = 'term' && permalink IS NOT NULL) AND taxonomy IN ($placeholders)
```

Comment:
> The &&, operator is a nonstandard MySQL extension. As of MySQL 8.0.17, this operator is deprecated and support for it will be removed in a future MySQL version. Applications should be adjusted to use the standard SQL AND operator.

See: https://dev.mysql.com/doc/refman/8.0/en/logical-operators.html#operator_and

```suggestion
			WHERE term_id NOT IN (SELECT object_id FROM $indexable_table WHERE object_type = 'term' AND permalink IS NOT NULL) AND taxonomy IN ($placeholders)
```

@herregroen It this worth creating an index on `permalink`?